code/modules/paperwork/paper.dm

```diff
@@ -476,6 +478,21 @@
 /obj/item/paper/crumpled/bloody
 	icon_state = "scrap_bloodied"
 
+/obj/item/paper/fortune
+	name = "fortune"
+	icon_state = "slip"
+	paper_height = 150
+	var/fortunemessage
```

Comment:
I need to remove this unused var when I next get a chance to (I think it was left over from an earlier iteration or something)